src/coreclr/src/jit/optcse.cpp

```diff
@@ -1578,25 +1578,25 @@ class CSE_Heuristic
         sortTab          = nullptr;
         sortSiz          = 0;
 
-#ifdef _TARGET_XARCH_
```

Comment:
This now removed:

    Removed increment of enregCount on _TARGET_X86_ when we have compLongUsed:
        Framework diffs
        Total PerfScoreUnits of diff: -654.75 (-0.00% of base)  diff is an improvement.
        79 total methods with Perf Score differences (55 improved, 24 regressed), 146432 unchanged.